src/test/java/lesson2/task5/IsEvenOrOddTest.java

```diff
@@ -0,0 +1,20 @@
+package lesson2.task5;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class IsEvenOrOddTest {
+
+    @Test
+    void verifyIfNumberIsEvenOrOdd() {
```

Comment:
It's better to split it into two separate test functions like: shouldVerifyNumberAsEven, shouldVerifyNumberAsNotEven :)

Reply:
In next lessons we will learn new ways of writing tests that will allow us to write one method and test it against plenty of variations of data. But For now, when you are testing two different cases is better to split them